### ISSUE TITLE
IMU Data Processing For Inertial Aided Navigation: A Recurrent Neural Network Based Approach. (arXiv:2103.14286v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#5202]
IMU Data Processing For Inertial Aided Navigation: A Recurrent Neural Network Based Approach. (arXiv:2103.14286v1 [cs.RO])<br>
https://ift.tt/2PDLt0A<br>
<p>In this work, we propose a novel method for performing inertial aided navigation, by using deep neural networks (DNNs). To date, most DNN inertial navigation methods focus on the task of inertial odometry, by taking gyroscope and accelerometer readings as input and regressing for integrated IMU poses (i.e., position and orientation). While this design has been successfully applied on a number of applications, it is not of theoretical performance guarantee unless patterned motion is involved. This inevitably leads to significantly reduced accuracy and robustness in certain use cases. To solve this problem, we design a framework to compute observable IMU integration terms using DNNs, followed by the numerical pose integration and sensor fusion to achieve the performance gain. Specifically, we perform detailed analysis on the motion terms in IMU kinematic equations, propose a dedicated network design, loss functions, and training strategies for the IMU data processing, and conduct extensive experiments. The results show that our method is generally applicable and outperforms both traditional and DNN methods by wide margins.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
